lib/taskana-core/src/test/java/acceptance/task/CancelTaskAccTest.java

```diff
@@ -0,0 +1,128 @@
+package acceptance.task;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import acceptance.AbstractAccTest;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import pro.taskana.common.api.exceptions.NotAuthorizedException;
+import pro.taskana.security.JaasExtension;
+import pro.taskana.security.WithAccessId;
+import pro.taskana.task.api.TaskService;
+import pro.taskana.task.api.TaskState;
+import pro.taskana.task.api.exceptions.InvalidStateException;
+import pro.taskana.task.api.exceptions.TaskNotFoundException;
+import pro.taskana.task.api.models.Task;
+import pro.taskana.task.api.models.TaskSummary;
+
+
+/** Acceptance tests for all claim and complete scenarios. */
+@ExtendWith(JaasExtension.class)
+class CancelTaskAccTest extends AbstractAccTest {
+  private TaskService taskService;
+
+  CancelTaskAccTest() {
+    super();
+    taskService = taskanaEngine.getTaskService();
+  }
+
+  @BeforeEach
+  public static void setupTest() throws Exception {
+    resetDb(false);
+  }
+
+  @WithAccessId(
+      userName = "user_1_1",
+      groupNames = {"group_1"})
+  @Test
+  void testQeryCancelledTasks() {
+    List<TaskSummary> taskSummaries =
+        taskService.createTaskQuery().stateIn(TaskState.CANCELLED).list();
+    assertThat(taskSummaries.size()).isEqualTo(5);
+  }
+
+  @WithAccessId(
+      userName = "admin",
```

Comment:
Is it a good idea to do all these tests as admin?

Reply:
Changed it ...